docs-gb/kubernetes/resources/model.md

```diff
@@ -9,8 +9,17 @@ A Model is the core atomic building block. It specifies a machine learning artif
 
 An example is shown below for a SKLearn model for iris classification:
 
-```{literalinclude} ../../../../../../samples/models/sklearn-iris-gs.yaml 
-:language: yaml
+```yaml
```

Comment:
Please reference the file via a @github-files/github-code-block. As much as possible, we want to change things in a single place, with the files in the repo being the source-of truth.

Reply:
unable to find the file

Reply:
Same as above I think the files are under v2/samples/{resource} so in this case: 
https://github.com/SeldonIO/seldon-core/blob/v2/samples/models/sklearn-iris-gs.yaml

Reply:
fixed